synapse/lib/storm.py

```diff
@@ -824,19 +824,47 @@
     {
         'name': 'pkg.list',
         'descr': 'List the storm packages loaded in the cortex.',
-        'cmdrargs': (),
+        'cmdargs': (
+            ('--verbose', {'default': False, 'action': 'store_true',
+                'help': 'Display build time for each package.'}),
+        ),
         'storm': '''
-            $pkgs = $lib.pkg.list()
-
-            if $($pkgs.size() = 0) {
+            init {
+                $conf = ({
+                    "columns": [
+                        {"name": "name", "width": 32},
+                        {"name": "vers", "width": 10},
+                    ],
+                    "separators": {
+                        "row:outline": false,
+                        "column:outline": false,
+                        "header:row": "#",
+                        "data:row": "",
+                        "column": "",
+                    },
+                })
+                if $cmdopts.verbose {
+                    $conf.columns.append(({"name": "time", "width": 20}))
+                }
+                $printer = $lib.tabular.printer($conf)
+            }
 
-                $lib.print('No storm packages installed.')
+            $pkgs = $lib.pkg.list()
 
-            } else {
+            if $($pkgs.size() > 0) {
                 $lib.print('Loaded storm packages:')
+                $lib.print($printer.header())
                 for $pkg in $pkgs {
-                    $lib.print("{name}: {vers}", name=$pkg.name.ljust(32), vers=$pkg.version)
+                    $row = (
+                        $pkg.name, $pkg.version,
+                    )
+                    if $cmdopts.verbose {
+                        $row.append($lib.time.format($pkg.build.time, '%Y-%m-%d %H:%M:%S'))
```

Comment:
I believe this will throw an exception if the data is missing. If so, we'll need to add a `try {} ... catch {}` block around it.